### PR TITLE
Added the coming soon and news section

### LINKED ISSUE #106: Add news and coming soon section
news and comingsoon
   
can i get assign to it?

### PR DESCRIPTION
**Pull request by Srijan1878**
#### Fixes #(issue number)
      Fixes #106  

 

#### Describe the changes you've made
Added the coming soon slider and also made a news section footer which has been termed as "coming soon"

#### Deploy your changes to github page or netlify and mention deployment link:
https://vigilant-hopper-0959a5.netlify.app/
Screenshot (352)
Screenshot (353)
Screenshot (354)

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

#### Test plan (OPTIONAL)
A good test plan should give instructions that someone else can easily follow.
How someone can test your code?

#### Checklist

- [x] I have mentioned related issue number in PR description.
- [x] I have added screenshots and related demo links of my changes..
- [x] The title of my pull request is a short description of the requested changes.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Open Source Program names(OPTIONAL: If you participated in any open-source program then please mention the program name here)LGMSOC

### CODE REVIEW

**Comment by Srijan1878** [comment]
This is a new PR for the coming soon page. I was facing some trouble while making changes for the previous PR, so I had to this.
I hope this time it's okay. @QAZIMAAZARSHAD

**Comment by QAZIMAAZARSHAD** [comment]
Your repo is not updated.
You have deleted the new navbar update.

Add a footer to the news page.

Remove the news slider from the home page.

**Comment by Srijan1878** [comment]
Please check the updates through the netlify link, I have updated it there too.

**Comment by QAZIMAAZARSHAD** [comment]
Check the scroll to the top button on the news page.

Rename new to "Coming Soon".

Remove unnecessary CSS you added in home.html.

Change content on the Coming soon page like rename 'Netflix', 'Hulu', to 'Apna Theatre'.

**Comment by Srijan1878** [comment]
Please have a look now @QAZIMAAZARSHAD

File: home.html [hunk 1]
```diff
@@ -50,6 +50,22 @@
     text-align: center;
 }
 
+.titleTextContainer{
```
Comment (QAZIMAAZARSHAD) [inline home.html:53]: @Srijan1878 I was talking about this CSS.

Is this of any use?

**Review by QAZIMAAZARSHAD — CHANGES_REQUESTED** [review]

**Comment by Srijan1878** [comment]
I removed the unused CSS of the slider which had been deleted, please have a look. @QAZIMAAZARSHAD

**Review by QAZIMAAZARSHAD — APPROVED** [review]

**Comment by QAZIMAAZARSHAD** [comment]
image

Make one PR to correct this.

**Pull request merged**